### ISSUE TITLE
Ordre chronologique non respecté dans le flux d'activité

### ISSUE THREAD

**Issue by jdesboeufs** [etalab/data.gouv.fr#28]
L'ordre chronologique des événements présents dans le flux d'activité n'est pas toujours respecté.

Voici un exemple avec mon profil : https://www.data.gouv.fr/fr/users/jerome-desboeufs/

<img width="861" alt="capture d ecran 2017-10-03 a 00 08 42" src="https://user-images.githubusercontent.com/1231232/31101710-274664dc-a7cf-11e7-8728-e25c14abc6ba.png">

**Comment by jdesboeufs** [comment]
J'ajoute que la puce devant l'icône du type d'événement me fait penser à un bug, mais je ne suis pas sûr.

**Comment by noirbizarre** [comment]
Je confirme, il y a 2 bugs sur cette capture d'écran !

**Comment by oncletom** [comment]
On dirait que opendatateam/udata#1059 et opendatateam/udata#1060 ont été revertés/pris dans les mailles de la suppression de `dev` :-(

**Comment by noirbizarre** [comment]
Effectivement, il faudrait les cherry-picker dans une PR sur le master actuel :/

**Comment by noirbizarre** [comment]
Bon, apparement si, ils sont bien embarqués (https://github.com/opendatateam/udata/blob/master/js/components/activities/timeline.vue#L164-L194) mais les quelsques tests que j'ai fait semble montrer qu'une fois compilé en mode prod' on perd le style :thinking:
